pytest_plugins/fixture_markers.py

```diff
@@ -6,16 +7,40 @@
 
 def pytest_generate_tests(metafunc):
     if 'rhel_contenthost' in metafunc.fixturenames:
-        rhel_parameters = [
-            dict(workflow=settings.content_host.deploy_workflow, rhel_version=ver)
-            for ver in settings.content_host.rhel_versions
-        ]
-        metafunc.parametrize(
-            'rhel_contenthost',
-            rhel_parameters,
-            ids=[f'rhel{r["rhel_version"]}' for r in rhel_parameters],
-            indirect=True,
-        )
+        rhel_parameters = []
+        for ver in settings.content_host.rhel_versions.keys():
+            # prefer nick-specific deploy workflow before using the default one
+            workflow = settings.content_host.rhel_versions[ver].get(
```

Comment:
Do a corresponding changes to `content_host.yaml.template` to show how rhel_versions now accept `deploy_workflow`!

Reply:
@jyejare when you check the PR descirption, you can see:
>merge preferably after:
>#9327

^ that PR introduces the changes you're requesting